src/pages/blog/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import { TGatsbyHead } from 'features/types'
+import { SEO } from 'components/containers'
+import { WithIntl } from 'components/localization'
+import PostsModule from 'features/pages/posts'
+
+const BlogPosts = ({ data }: TBlogPosts) => {
+    return <PostsModule data={data} />
+}
+
+export default WithIntl()(BlogPosts)
+
+export const query = graphql`
+    query {
+        allStrapiPost(limit: 10) {
```

Comment:
Also can add sorting descending order, so the latest posts shows on top
```suggestion
        allStrapiPost(limit: 10) {
```